webcommon/javascript2.kit/src/org/netbeans/modules/javascript2/kit/Bundle.properties

```diff
@@ -20,4 +20,4 @@ OpenIDE-Module-Name=JavaScript2 Kit
 OpenIDE-Module-Short-Description=An umbrella module covering all modules required for JavaScript support.
 OpenIDE-Module-Long-Description=\
     JavaScript support: editing, refactoring, hints, etc.\n\n\
-    For more information, see\nhttp://wiki.netbeans.org/wiki/view/JavaScript
+    For more information, see\n <a href="https://netbeans.apache.org/wiki/main/wiki/JavaScript/">Apache NetBeans Javascript</a>
```

Comment:
link doesn't work: https://netbeans.apache.org/wiki/main/wiki/JavaScript/

nitpick: 
 - the `\n` is probably no longer needed before the link
 - change Javascript to JavaScript